src/developer-quickstart.md

```diff
@@ -0,0 +1,441 @@
+    // Now you have an instance of your contract you can use to test each function
+    instance.increment().call().await.unwrap();
+    let result = instance.count().call().await.unwrap();
+    assert!(result.value > 0); 
+}
+```
+
+Run the following command in the terminal: `forc test`.
+
+You'll see something like this as your output:
+
+```console
+ Compiled library "core".
+  Compiled library "std".
+  Compiled contract "counter_dapp".
+  Bytecode size is 208 bytes.
+  Compiling counter+dapp v0.1.0 (</path/to/counter_dapp>)
+  Finished test [unoptimized + debuginfo] target(s) in 11.71s
+  Running tests/harness.rs (target/debug/deps/integration_tests-6a600a6a87f48edb)
+
+running 1 test
+test can_get_contract_id ... ok
+
+test result: ok. 1 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out; finished in 0.24s
+```
+
+## Deploy the Contract
+
+It's now time to deploy the contract and call it on a Fuel node. We will show how to do this using `forc` from the command line, but you can also do it using the [Rust SDK](https://github.com/FuelLabs/fuels-rs#deploying-a-sway-contract) or the [TypeScript SDK](https://github.com/FuelLabs/fuels-ts/#deploying-contracts).
+
+We'll be deploying to the Fuel testnet. In order to deploy a contract, you need to have a wallet to sign the transaction. You can optionally pass the flag `--unsigned` to deploy a contract without a signature.
+
+By default fuel-core runs without UTXO validation, which means you can run unsigned transactions. This allows you to send invalid inputs to emulate different conditions.
+
+### Install the Wallet CLI
+
+Follow [these steps to set up a wallet to sign transactions](https://github.com/FuelLabs/forc-wallet#forc-wallet).
+
+After typing in a password, be sure to save the mnemonic phrase that is output.
+
+With this, you'll get a fuel address that looks something like this: `fuel1efz7lf36w9da9jekqzyuzqsfrqrlzwtt3j3clvemm6eru8fe9nvqj5kar8`. Save this address as you'll need it to sign transactions when we deploy the contract.
+
+#### Deploy To Testnet
+
+Now that you have a wallet, you can deploy with `forc deploy` and passing in the testnet endpoint like this:
+
+`forc deploy --url https://node-beta-1.fuel.network/graphql --gas-price 1`
+
+> Note: We set the gas price to 1. Without this flag, the gas price is 0 by default and the transaction will fail.
+
+The terminal will ask for the address of the wallet you want to sign this transaction with, paste in the address you saved earlier, it looks like this: `fuel1efz7lf36w9da9jekqzyuzqsfrqrlzwtt3j3clvemm6eru8fe9nvqj5kar8`
+
+The terminal will output your contract id. Be sure to save this as you will need it to build a frontend with the Typescript SDK.
+
+The terminal will output a `transaction id to sign` and prompt you for a signature. Open a new terminal tab and view your accounts by running `forc wallet list`. If you followed these steps, you'll notice you only have one account, `0`.
```

Comment:
As we ask the user to save the wallet address before we could remove the need to list addresses from here, and just mention get the address and use it.